lib/logstash/outputs/elasticsearch.rb

```diff
@@ -203,14 +202,16 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # How long to wait, in seconds, between sniffing attempts
   config :sniffing_delay, :validate => :number, :default => 5
 
-  # Set max retry for each event
+  # Set max retry for each event. The total time spent blocked on retries will be
+  # (max_retries * retry_max_interval). This may vary a bit if Elasticsearch is very slow to respond
   config :max_retries, :validate => :number, :default => 3
 
-  # Set retry policy for events that failed to send
-  config :retry_max_items, :validate => :number, :default => 5000
+  # Set max interval between bulk retries.
+  config :retry_max_interval, :validate => :number, :default => 2
 
-  # Set max interval between bulk retries
-  config :retry_max_interval, :validate => :number, :default => 5
+  # DEPRECATED This setting no longer does anything. If you need to change the number of retries in flight
+  # try increasing the total number of workers to better handle this.
+  config :retry_max_items, :validate => :number, :default => 5000, :deprecated => true
```

Comment:
Make this 500, even though its deprecated? We missed this in 2.0 and is out of sync with `flush_size`